normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
+          "emoji": "\uD83D\uDFE8"
+        },
+        {
+          "name": "low",
+          "color": "POSITIVE",
+          "description": "Issue doesn't hurt anyone",
+          "emoji": "\uD83D\uDFE9"
+        }
+      ]
+    },
+    {
+      "name": "status",
+      "labels": [
+        {
+          "name": "ready for dev",
+          "color": "LIGHT",
+          "description": "Issue is ready for work as blockers have been resolved",
+          "emoji": "\uD83C\uDFC1"
+        },
+        {
+          "name": "blocked",
+          "color": "MEDIUM",
+          "description": "Issue is not ready for work due to a PR or issue blocking it",
```

Comment:
Blockers may not just be a PR or issue; e.g. we may need to coordinate with a third-party. It's probably best not to specify what the blocker is.

Reply:
Updated in caf892e.